classes/admin.php

```diff
@@ -80,20 +95,23 @@ class Object_Sync_Sf_Admin {
 	* @param object $pull
 	* @param object $logging
 	* @param array $schedulable_classes
+	* @param object $queue
 	* @throws \Exception
 	*/
-	public function __construct( $wpdb, $version, $login_credentials, $slug, $wordpress, $salesforce, $mappings, $push, $pull, $logging, $schedulable_classes ) {
+	public function __construct( $wpdb, $version, $login_credentials, $slug, $option_prefix, $wordpress, $salesforce, $mappings, $push, $pull, $logging, $schedulable_classes, $queue ) {
```

Comment:
Since `__construct` is a public method and this is the 1.4 release (not a 2.0 backwards compatibility breaking release), the new parameters should be optional. We can't guarantee developers haven't invoked this method. (As unlikely as that seems for an admin pages class.)

Reply:
So instead of adding `$option_prefix` to the arguments where it might make the most sense, do you think I should instead put it after all the pre-existing requirements to prevent this? On these classes where that applies, I mean.